tests/testthat/test-signal-deprecated.R

```diff
@@ -9,62 +9,90 @@ test_that("signallers require function call syntax for `what`", {
 })
 
 test_that("deprecation messages are constructed for functions", {
-  expect_known_output(file = test_path("output", "test-signal-message.txt"), {
-    cat_ruler("Inferred package name (here it is base b/c of testthat's eval env)")
-    cat_line(lifecycle_build_message("1.0.0", "foo()", signaller = "deprecate_stop"))
-
-    cat_ruler("Overridden package name")
-    cat_line(lifecycle_build_message("1.0.0", "mypkg::foo()", signaller = "deprecate_stop"))
-
-    cat_ruler("Replacement function")
-    cat_line(lifecycle_build_message("1.0.0", "foo()", "bar()", signaller = "deprecate_stop"))
-
-    cat_ruler("Replacement function with overridden package names (1)")
-    cat_line(lifecycle_build_message("1.0.0", "foo::quux()", "foofy()", signaller = "deprecate_stop"))
-
-    cat_ruler("Replacement function with overridden package names (2)")
-    cat_line(lifecycle_build_message("1.0.0", "foo::quux()", "bar::foofy()", signaller = "deprecate_stop"))
-
-    cat_ruler("Details")
-    details <- glue::glue(
-      "
+  expect_snapshot({
+    # https://github.com/r-lib/rlang/issues/1087
+    NULL
+
+    {
```

Comment:
Why are you using `{` here?

Reply:
So there's only one `Code:` section per test in the snapshot. Otherwise each expression gets one section and the snapshot file is hard to read.

Reply:
Oh hmmmm, then maybe that's a bug in testthat?